plugins/SitesManager/API.php

```diff
@@ -394,6 +394,22 @@ public function getSitesWithAdminAccess($fetchAliasUrls = false, $pattern = fals
         return $sites;
     }
 
+    /**
+     * Returns the messages to warn users on site deletion.
+     *
+     * @throws Exception if the website ID doesn't exist or the user doesn't have access to it
+     * @param int $idSite
+     * @return string messages to warn users
+     */
+    public function getMessagesToWarnOnSiteRemoval($idSite)
```

Comment:
Please try to use type hints where possible

```suggestion
    public function getMessagesToWarnOnSiteRemoval(int $idSite): array
```